content/posts/newsletter-013/index.md

```diff
@@ -29,7 +29,8 @@ Table of contents:
 
 - [Game Updates](#game-updates)
 - [Learning Material Updates](#learning-material-updates)
-- [Library & Tooling Updates](#library-tooling-updates)
+- [Library & Tooling Updates](#library--tooling-updates)
+  - [Bevy Engine](#bevy-engine)
```

Comment:
To keep it from being super long, we don't usually add the individual sections to the TOC.
```suggestion
```

Reply:
oops i didn't manually add this / this wasn't my intention. looks like my markdown vscode plugin manages table of contents automatically:

![image](https://user-images.githubusercontent.com/2694663/92179398-72fc5300-edf9-11ea-85f9-757c202c1964.png)

Sorry about that! I should have looked at the diff more carefully.

Reply:
Someone else had the same thing happen in their PR, it's no problem :)